package.json

```diff
@@ -69,6 +69,7 @@
     "web-vitals": "^2.1.4"
   },
   "devDependencies": {
+    "@dcl/eslint-config-dapps": "github:decentraland/eslint-config-dapps",
```

Comment:
Currently, we're using our own repo for rules.

tbd if we'll use the `@dcl/eslint-config` repo to store `dapps` rules or wait for npm registry access.